src/status_im/contexts/communities/actions/permissions_sheet/view.cljs

```diff
@@ -0,0 +1,34 @@
+(ns status-im.contexts.communities.actions.permissions-sheet.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [status-im.contexts.communities.actions.permissions-sheet.style :as style]
+    [status-im.contexts.communities.utils :as communities.utils]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn view
+  [id]
+  (let [{:keys [permissions]} (rf/sub [:community/token-permissions id])
+        role-text             (fn [role]
```

Comment:
this could be defined outside, as it will be recreated at each render (I don't think it matters, but it also looks a bit neater to defined it outside, since `let` to me implies is a closure